decTest/exp.decTest

```diff
@@ -540,6 +540,7 @@ expx1200 exp 1 -> 2.718281828459045235360287471352662 Inexact Rounded
 Precision: 34
 maxExponent: 6144
 minExponent: -6143
+clamp: 1

```

Comment:
Why do we need this? This test should not change at all.

Reply:
It does because of these lines in the run-test script:

```rust
    let d128_env = Environment {
        precision: Some(34),
        rounding: Some(Rounding::HalfEven),
        max_exponent: Some(6144),
        min_exponent: Some(-6143),
        extended: true,
        clamp: true,
    };

    if *env != d128_env {
        return TestResult::Ignored(test);
    }
```

Without that change, the test was silently skipped. 

Maybe I should have gone in the other direction and defined a different Environment just for these tests? I assume that the check is there to make sure that it only runs applicable tests, but since the exp tests pass with the different clamping mode, I assume that all of those inputs and outputs are still valid.

Reply:
Is it possible to change `run_test()` to run ops with the specified environment which will make this change unnecessary and increase test coverage?